src/settings/lib/RuleEditor/Rules-hint.js

```diff
@@ -63,7 +64,7 @@ function getItemOptions(selector, typeKey) {
   const displayText = getDisplayText(selector, typeKey);
 
   return {
-    text: `${text} `,
+    text: addIndentToCMRules(text, 'after'),
```

Comment:
Could you please clarify what CM means?

Reply:
Sure. CM stands for [CodeMirror](https://codemirror.net/) - the name of text editor which is used as Circulation Rules Editor.

Reply:
I think it would be better to omit this in the name for better readability or write as you mentioned it `CodeMirror`.

Reply:
Agree. Changed func name to `addIndentToEditorRules`.